### DISCUSSION TITLE
Search feature in github code viewer for ios

### DISCUSSION THREAD

**Discussion by kugesh-Rajasekaran** [community/community#41162] [Mobile] [answered]
Hi... when we view code in GitHub mobile (repository files), there is no searching functionality available to search variable, method names etc.., It would be very nice if we can able to search it with keywords of what we are looking for.

**Comment by candyho** [comment, marked as answer]
Hey thanks for sharing the feedback! I've logged this request in our internal repo for tracking purpose. Hopefully we can tackle this in the near future.

**Comment by Guilhermeeidt33** [comment]
Senha intagran
